Makefile.am

```diff
@@ -189,7 +189,15 @@ $(BITCOIN_WALLET_BIN): FORCE
 	$(MAKE) -C src $(@F)
 
 if USE_LCOV
-LCOV_FILTER_PATTERN=-p "/usr/include/" -p "/usr/lib/" -p "src/leveldb/" -p "src/bench/" -p "src/univalue" -p "src/crypto/ctaes" -p "src/secp256k1"
+LCOV_FILTER_PATTERN = \
+	-p "/usr/include/" \
+	-p "/usr/lib/" \
```

Comment:
Could you add a commit to also exclude lib64, please. This shows up when compiling with clang-8 for example:

https://pbs.twimg.com/media/ELKJFh3XsAAM6mJ?format=png&name=medium

Reply:
Done! Guessing that I should squash the commits?

Reply:
I think it's fine to have these as separate commits, they're not really related

Reply:
Sounds good, will keep it as is! 👌